### PR TITLE
adding mof file for freb log

### PR DESCRIPTION
**Pull request by pan-wang**

### CODE REVIEW

File: src/AspNetCore/AspNetCore.vcxproj [hunk 1]
```diff
@@ -244,12 +244,16 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="ancm.mof" />
```
Comment (jkotalik) [inline src/AspNetCore/AspNetCore.vcxproj]: Do we need a mof file for the request handler?

File: src/AspNetCore/ancm.mof [hunk 2]
```diff
@@ -0,0 +1,208 @@
+#pragma classflags("forceupdate")
+#pragma namespace ("\\\\.\\Root\\WMI")
+#pragma autorecover
+
+/*
+ * AspNetCore module trace events layout
+ * Uncomment the following class to run mof2trace to generate header file
+ * comment it back before checking it in */
```
Comment (jkotalik) [inline src/AspNetCore/ancm.mof]: Do we need to do what the comment says?

File: src/AspNetCore/ancm.mof [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    [WmiDataId(2),
+     Description("Application Start Failure Description") : amended,
+     StringTermination("NullTerminated"),
+     format("w"),
+     read]
+     string FailureDescription;
+};
+
+[Dynamic,
+ Description("Start fardwarding request") : amended,
```
Comment (jkotalik) [inline src/AspNetCore/ancm.mof]: nit "Started forwarding the request"

**Comment by shirhatti** [comment]
Do you need to regenerate aspnetcore_event.h using mof2trace now that you've modified the mof file?

**Pull request merged**